src/view/ChewingEditor.cpp

```diff
@@ -190,11 +199,16 @@ void ChewingEditor::setupFileSelection()
 
 void ChewingEditor::setupImport()
 {
+    // initialize importCounter_
+    importedCounter_ = 0;
+
     shortcut_import_ = new QShortcut(Qt::CTRL + Qt::Key_I, this);
     connect(
         shortcut_import_, SIGNAL(activated()),
         this, SLOT(selectImportFile())
     );
+    // initialize importCounter_
+    importedCounter_ = 0;
```

Comment:
Please avoid inconsistent naming `importedCounter_`, which consists of [camelCase](https://en.wikipedia.org/wiki/CamelCase) and suffixing underscore.